packages/client/modules/teamDashboard/components/TeamDashInsights/TeamInsightsCard.tsx

```diff
@@ -0,0 +1,77 @@
+import graphql from 'babel-plugin-relay/macro'
+import React, {ReactNode, useState} from 'react'
+import {useFragment} from 'react-relay'
+import Tooltip from '../../../../components/Tooltip'
+import {Info as InfoIcon} from '@mui/icons-material'
+import {ThumbUp, ThumbDown} from '@mui/icons-material'
+import FlatButton from '../../../../components/FlatButton'
+import SendClientSegmentEventMutation from '../../../../mutations/SendClientSegmentEventMutation'
+import useAtmosphere from '../../../../hooks/useAtmosphere'
+import {TeamInsightsCard_insights$key} from '../../../../__generated__/TeamInsightsCard_insights.graphql'
+import TeamInsightsId from '../../../../shared/gqlIds/TeamInsightsId'
+import clsx from 'clsx'
+
+interface Props {
+  title: string
+  tooltip: string
+  children: ReactNode
+  teamInsightsRef: TeamInsightsCard_insights$key
+}
+
+const TeamInsightsCard = (props: Props) => {
+  const {children, teamInsightsRef, title, tooltip} = props
+  const [isHelpful, setIsHelpful] = useState<boolean>()
+
+  const insights = useFragment(
+    graphql`
+      fragment TeamInsightsCard_insights on TeamInsights {
+        id
+      }
+    `,
+    teamInsightsRef
+  )
+
+  const {id} = insights
+  const teamId = TeamInsightsId.split(id)
+
+  const atmosphere = useAtmosphere()
+  const {viewerId} = atmosphere
+
+  const trackClick = (isHelpfulInsight: boolean) => {
+    SendClientSegmentEventMutation(atmosphere, title, {
+      viewerId,
+      teamId,
+      isHelpfulInsight
+    })
```

Comment:
@tghanken I added the isHelpfulInsight property to the segment events and then just send the title of the card. My goal here is to reuse this component for other insight cards as well. What do you think?

Reply:
-1 I'd recommend going with something like this instead.  We want the event itself (Insight Card Feedback Clicked) to be the same across all the various insights cards, then to use properties to segment between the individual types.  My recommended component should be safe to reuse across all insights cards!

```suggestion
    SendClientSegmentEventMutation(atmosphere, 'Insight Card Feedback Clicked', {
      viewerId,
      teamId,
      cardName: title,
      isHelpfulInsight
    })
```